From 3ccb71eeca42dbcd5e4d00ae1877a489ae82598d Mon Sep 17 00:00:00 2001
From: Yu Chien Peter Lin <peterlin@andestech.com>
Date: Wed, 29 Dec 2021 16:04:54 +0800
Subject: [PATCH] Disable PIC explicitly for assembling

This patch is necessary if the fw_dynamic load address
is not equal to link address.
However, they are equal currently, since we include an u-boot
patch for preventing fw_dynamic relocation.

Signed-off-by: Yu Chien Peter Lin <peterlin@andestech.com>
---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index d6f097d..441518d 100644
--- a/Makefile
+++ b/Makefile
@@ -225,6 +225,7 @@ ASFLAGS		+=	-mcmodel=$(PLATFORM_RISCV_CODE_MODEL)
 ASFLAGS		+=	$(GENFLAGS)
 ASFLAGS		+=	$(platform-asflags-y)
 ASFLAGS		+=	$(firmware-asflags-y)
+ASFLAGS		+=	-fno-pic

 ARFLAGS		=	rcs

--
2.25.1
